src/components/Proposal/Create/SchemeForms/CreateContributionRewardProposal.tsx

```diff
@@ -290,6 +302,7 @@ class CreateContributionReward extends React.Component<IProps, IStateProps> {
                     name="beneficiary"
                     onBlur={(touched) => { setFieldTouched("beneficiary", touched); }}
                     onChange={(newValue) => { setFieldValue("beneficiary", newValue); }}
+                    defaultValue={importUrlValues(initialFormValues).beneficiary}
```

Comment:
if you do the importUrlValues when you setup initialFormValues one time in the constructor then you dont have to call it multiple times